### PR TITLE
Spring cleanups

### PR DESCRIPTION
**Pull request by rejas**
Cleanups all over the place:
- dependencies 
- typos
- imports
- == --> ===
- missing ; and vars

### CODE REVIEW

**Comment by rejas** [comment]
I think I am finished for now @MichMich I hope its not too messy for you to review, I tried to keep the cleanups simple and self explanatrory.

File: translations/sv.json [hunk 1]
```diff
@@ -25,7 +25,6 @@
 	"WNW": "VNV",
 	"NW": "NV",
 	"NNW": "NNV",
-	"FEELS": "Känns som",
```
Comment (MichMich) [inline translations/sv.json]: Just curious: why is this translation removed?

Comment (rejas) [reply translations/sv.json]: Because its also in line 35 :-)

Comment (MichMich) [reply translations/sv.json]: Ah, yes. GitHub didn't show that! :)

**Comment by MichMich** [comment]
Merged!

**Pull request merged**